components/MapComponent.tsx

```diff
@@ -1,19 +1,27 @@
+import { ImageSource } from "aws-sdk/clients/lookoutvision"
 import { Location } from "lib/location/Location"
 import React, { useState } from "react"
-import { StyleProp, View, ViewStyle } from "react-native"
-import MapView, { Marker, PROVIDER_GOOGLE } from "react-native-maps"
+import { StyleProp, StyleSheet, View, ViewStyle } from "react-native"
+import MapView, { Circle, Marker, PROVIDER_GOOGLE } from "react-native-maps"
 
 // Type for the map markers, in order to separate their logic out
-export interface mapMarker {
+export interface MapMarker {
   key: string
   title: string
   location: Location
 }
 
+export interface MarkerCustomize {
```

Comment:
Are there any other style props we can define for the markers? Icon, pin color and circle color should be determined from the event data rather than defined here.